### PR TITLE
Update README section with the local endpoints

### PR DESCRIPTION
**Pull request by dmitryax**
Reword it a bit and add missing Prometheus endpoint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,9 +39,13 @@ docker compose up
...
 
-- Once the images are built you can access the Webstore at: <http://localhost:8080>
+Once the images are built and containers are started you can access:
 
-- And the Jaeger UI at: <http://localhost:16686>
+- Webstore: <http://localhost:8080>
+
+- Jaeger: <http://localhost:16686>
+
+- Prometheus: <http://localhost:9090>
```
Comment (reyang) [inline README.md:48]:
```suggestion
- Webstore: <http://localhost:8080/>

- Jaeger: <http://localhost:16686/>

- Prometheus: <http://localhost:9090/>
```

**Review by reyang — APPROVED** [review]
LGTM.

**Comment by cartersocha** [comment]
Nice. Thanks for the clean up

**Comment by cartersocha** [comment]
& welcome to the project 🚀

**Pull request merged**